logging/logrus/middleware.go

```diff
@@ -86,6 +87,7 @@ func newServerRequestFields(req *http.Request) logrus.Fields {
 		"http.request.user_agent":   req.Header.Get("User-Agent"),
 		"http.request.length_bytes": req.ContentLength,
 		"http.request.referer":      req.Referer(),
+		"http.start_time":           startTime,
```

Comment:
`Format(time.RFC3339)`?

Reply:
If we use `time.Format`, we have to then Parse it from the Hook, which could (?) yield an un-handable error. I'm not opposed to it but `Time` already has a `String()` function which is mostly human readable.